brontide/conn.go

```diff
@@ -287,3 +287,10 @@ func (c *Conn) RemotePub() *btcec.PublicKey {
 func (c *Conn) LocalPub() *btcec.PublicKey {
 	return c.noise.localStatic.PubKey()
 }
+
+// ClearPendingSend drops references to the next header and body buffers so
+// that the memory can be garbage collected.
+func (c *Conn) ClearPendingSend() {
+	c.noise.nextHeaderSend = nil
```

Comment:
Does it make sense to have this as a separate commit, since we replace this by a call to `c.noise.releaseBuffers()` in the next commit. I would consider (merging into the next/dropping this) commit.